devito/finite_differences/operators.py

```diff
@@ -54,9 +57,12 @@ def grad(func, shift=None, order=None, method='FD'):
     method: str, optional, default='FD'
         Discretization method. Options are 'FD' (default) and
         'RSFD' (rotated staggered grid finite-difference).
+    weights/w: list, tupe, or dict, optional, default=None
```

Comment:
Looks like the typo was copied into all the docstrings

Reply:
Fixed